docs/source/tagset/elemento-etal.rst

```diff
@@ -13,14 +13,11 @@ Ocorre:
   Zero ou uma vez
 
 
-Elemento utilizado quando há mais de três autores, sendo que somente se indica o
-primeiro, acrescentando-se a expressão ``et al.``, que significa "entre outros".
+Elemento utilizado quando há mais de três autores, sendo que somente se indica o primeiro, acrescentando-se a expressão ``et al.``, que significa "entre outros".
 
```

Comment:
alterei o início da frase para 'Elemento utilizado com a finalidade de indicar contribuidores não identificados, sendo que...' e o resto ficou igual.